recipes/ffgc/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "2.4.1" %}
+
+package:
+  name: ffgc
+  version: "{{ version }}"
+
+source:
+  url: "https://gitlab.ub.uni-bielefeld.de/gi/FFGC/-/archive/{{ version }}/FFGC-{{ version }}.tar.bz2"
+  md5: a4e71dc8053b595527e447807b46646e
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - "echo $PREFIX"
+    - "mkdir -p $PREFIX/bin/"
+    - "mkdir -p $PREFIX/opt/ffgc/"
+    - "cp -r * $PREFIX/opt/ffgc/"
+    - "chmod u+x $PREFIX/opt/ffgc/ffgc_create_project.py"
+    - "ln -s $PREFIX/opt/ffgc/ffgc_create_project.py $PREFIX/bin/"
+
+requirements:
+  host:
+    - python >=3.7  # then also goes to run requirements
+  run:
+    - snakemake >=7.24
```

Comment:
add:
```yaml
 - python
 ```
 
 or 
 
 ```yaml
  - python =3.11
 ```
 
 to get around the current bug until the bioconda devs fix it

Reply:
Still got the same error with =3.11, will try just python.